src/Common/src/System/Collections/HashHelpers.cs

```diff
@@ -50,7 +44,6 @@ public static int GetPrime(int min)
         {
             if (min < 0)
                 throw new ArgumentException(SR.Arg_HTCapacityOverflow);
-            Contract.EndContractBlock();
```

Comment:
typo below
            // Allow the hashtables to grow to maximum possible size (~2G elements) before **encoutering** capacity overflow.